iwyu_include_picker.cc

```diff
@@ -1303,15 +1303,15 @@ vector<string> IncludePicker::GetCandidateHeadersForFilepath(
 vector<string> IncludePicker::GetCandidateHeadersForFilepathIncludedFrom(
     const string& included_filepath, const string& including_filepath) const {
   vector<string> retval;
-  const string quoted_includer = ConvertToQuotedInclude(including_filepath);
-  const string quoted_includee = ConvertToQuotedInclude(included_filepath);
+  const string quoted_includer = ConvertToQuotedInclude(including_filepath, MakeAbsolutePath(GetParentPath(including_filepath)));
```

Comment:
`quoted_includer` is only used to search the friends map. This map looks a bit strange to me. IMO the keys should be absolute paths but I think they aren't. I just included this change to have it consistent. It is basically the include as-if it would be included. (includer including itself, sounds kinda confusing, but I think you get the point)